server/src/core/server/config.ts

```diff
@@ -505,6 +505,18 @@ const config = convict({
     default: ms("3000s"),
     env: "NOTIFICATIONS_POLL_RATE",
   },
+  external_notifications_api_url: {
+    doc: "URL to forward notifications information to an external url.",
+    format: "url",
+    default: null,
+    env: "EXTERNAL_NOTIFICATIONS_API_URL",
+  },
+  external_notifications_api_key: {
+    doc: "API key to use when forwarding notifications to an external url",
+    format: String,
+    default: null,
```

Comment:
I think this needs to have a default of empty string instead of null, as otherwise there is an error when starting up the server if nothing is set for the external notifications api key. Same note for the api key; it also seems to need a default url.

I guess then the check in the notifications service to see if it's active needs to be updated too to check for not empty string, etc.

Reply:
I've set this to the default url of `http://localhost:7003/api`, that should stop the errors when this isn't set in the env vars

this also works for the active state because you also have to set a valid API key for the service to become active 🙂